### PR TITLE
Add if_versions tags so windows support does not show up in KM v2.4

### PR DESCRIPTION
**Pull request by johnharris85**
### Description

Remove Windows support from KM 2.4 onwards

### Testing instructions

Netlify link: 

### Checklist 

- [x] Review label added 
- [ ] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Comment by johnharris85** [comment]
> @johnharris85 , just confirming you want this change pushed ahead of the 2.4 docs release? or as part of that release?

Ye it can go now please :)

File: app/_src/mesh/features/windows.md [hunk 1]
```diff
@@ -94,3 +94,4 @@ but you can use a persistent storage like PostgreSQL by updating the `conf/kuma-
 [backends]: https://kuma.io/docs/1.8.x/documentation/configuration/
 {% endif_version %}
 
+{% endif_version %}
```
Comment (Guaris) [inline app/_src/mesh/features/windows.md:97]:
```suggestion
```

File: app/_src/mesh/features/windows.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Kong Mesh with Windows
 ---
-
+{% if_version lte:2.3.x %}
```
Comment (Guaris) [inline app/_src/mesh/features/windows.md:4]:
```suggestion
```

File: app/_src/mesh/installation/windows.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Kong Mesh with Windows
 ---
-
+{% if_version lte:2.3.x %}
```
Comment (Guaris) [inline app/_src/mesh/installation/windows.md:4]:
```suggestion
```

File: app/_src/mesh/installation/windows.md [hunk 4]
```diff
@@ -88,3 +88,4 @@ but you can use a persistent storage like PostgreSQL by updating the `conf/kuma-
 [backends]: https://kuma.io/docs/1.8.x/documentation/configuration/
 {% endif_version %}
 
+{% endif_version %}
```
Comment (Guaris) [inline app/_src/mesh/installation/windows.md:91]:
```suggestion
```

**Review by Guaris — COMMENTED** [review]
In 2.4 we just wont include the windows files in the nav bar.

**Review by Guaris — APPROVED** [review]
LGTM Merging after tests pass

**Pull request merged**